content/it/community/_index.html

```diff
@@ -1,69 +1,237 @@
-          </div>
-        <div class="content">
-            <h3>Special Interest Groups (SIGs)</h3>
-            <p>hai un interesse particolare nel modo in cui Kubernetes funziona con un'altra tecnologia? Guarda la nostra crescita
-                <a href="https://git.k8s.io/community/sig-list.md">lista dei SIG</a>,
-                da AWS e Openstack a Big Data e Scalabilità, c'è un posto per te per contribuire e istruzioni
-                puoi creare un nuovo SIG di tuo interesse se non e presente(ancora).</p>
-
-            <p>Come membro della comunità di Kubernetes, puoi partecipare a qualsiasi riunione SIG
-                a cui sei interessato. Non è richiesta alcuna registrazione.</p>
-
-        </div>
-
-        <div class="content">
-          <h3>Codice di condotta</h3>
-          <p>La comunità di Kubernetes apprezza il rispetto e l'inclusività, e
-             applica un <a href="code-of-conduct/">Codice di condotta</a> in tutte
-             le interazioni. Se si nota una violazione del Codice di condotta a
-             in un evento o meeting in Slack, or in qualsiasi canale di comunicazione
-             contatta il <a href="https://github.com/kubernetes/community/tree/master/committee-code-of-conduct">Comitato di Condotta di Kubernetes</a>
-             <a href="mailto:conduct@kubernetes.io">conduct@kubernetes.io</a>.
-             Il tuo anonimato sarà protetto.
-          </p>
-         </div>
-    </main>
-</section>
-
-<section id="talkToUs">
-    <main>
-        <h3>Parla con noi!</h3>
-        <h4>Ci piacerebbe sapere da te, come stai usando Kubernetes,<br> e cosa possiamo fare per renderlo migliore.</h4>
-        <div id="bigSocial">
-            <div>
-                <a href="https://twitter.com/kubernetesio">@kubernetesio</a>
-                <p>Ricevi le ultime notizie e aggiornamenti.</p>
-            </div>
-            <div>
-                <a href="https://github.com/kubernetes/kubernetes">GitHub Project</a>
-                <p>Guarda il progetto e considera di contribuire</p>
-            </div>
-            <div>
-                <a href="http://slack.k8s.io/">#kubernetes-users</a>
-                <p>Il nostro canale Slack è il modo migliore per contattare i nostri ingegneri e condividere le tue idee con loro.</p>
-            </div>
-            <div>
-                <a href="http://stackoverflow.com/questions/tagged/kubernetes">Stack Overflow</a>
-                <p>Il nostro forum di utenti è un ottimo posto dove andare per il supporto della comunità.</p>
-            </div>
-        </div>
-    </main>
-</section>
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="mobile">
```

Comment:
Optional fix: localize the `alt` attributes.

Reply:
thanks